packages/opencensus-web-exporter-ocagent/test/test-ocagent.ts

```diff
@@ -107,7 +107,11 @@ describe('OCAgentExporter', () => {
       node: {
         identifier: {hostName: window.location.host},
         serviceInfo: {name: 'testService'},
-        libraryInfo: {exporterVersion: '0.0.1'},
+        libraryInfo: {
+          language: 10,
+          coreLibraryVersion: '0.0.1',
```

Comment:
Consider using `VERSION` and `EXPORTER_VERSION` instead of hard code. Otherwise this test needs to be updated on every new release.

Reply:
Good point! Updated to use the constants for the versions.